configs/babel/targets.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   node: {
-    node: '12'
+    node: '18'
```

Comment:
lowered target to 18 for older compatibility

Reply:
why not use TLS which is 20? Older node versions can always point to older radpack

Reply:
@asilvas-godaddy 18 is still considered an LTS version, and since this is only about transcompiling src down to lib/dist assets, keeping the node compatibility for the lowest maintained version seems like a good choice for OSS.